config/pipelines/high_throughput_scrna_core_cell_extraction.wip.yml

```
@@ -0,0 +1,21 @@
+---
+# Initial transfer from tube to plate, as entry for below pipeline. Happens without submissions / requests.
+scRNA Core Cell Extraction Entry:
+  pipeline_group: scRNA Core Cell Extraction Cell Banking
+  relationships:
+    LRC Blood Vac: LRC Blood Bank
+# scRNA Core Cell Extraction cell banking pipeline
+scRNA Core Cell Extraction Banking Seq:
```

```suggestion
scRNA Core Cell Extraction Seq:
```
Sorry, a few similar comments here. We switched from calling this "scRNA Core Cell Banking" because actually the isolations for proper cell banking are being done in faculty... it makes more sense to just call this "scRNA Core Cell Extraction", and leave out the "banking" terminology.